homeassistant/components/homekit/const.py

```diff
@@ -5,6 +5,12 @@
 HOMEKIT_FILE = '.homekit.state'
 HOMEKIT_NOTIFY_ID = 4663548
 
+# #### BATTERY SERVICE ####
+ATTR_CHARGING_STATE = 'charging_state'
+ATTR_CHARGING_STATE_CHARGING = 'charging'
+ATTR_CHARGING_STATE_NOT_CHARGING = 'not_charging'
+ATTR_CHARGING_STATE_NOT_SUPPORTED = 'not_supported'
```

Comment:
Those should go inside `homeassistant.const` to be accessible for other platforms and components.

Reply:
Removed charging state tracking.